src/sentry/static/sentry/app/views/issueList/container.jsx

```diff
@@ -32,6 +32,11 @@ class IssueListContainer extends React.Component {
       ? 'cold-start'
       : 'warm-start';
     metric.mark({name: 'page-issue-list-start', data: {start_type: startType}});
+    metric.createSpan({
```

Comment:
When you were testing this, did you make sure that there aren't other spans that finish before this? If this starts before any other spans finish then it should work to capture spans without triggering the idleTimeout code.

Reply:
yup seems like this span starts too late to prevent idleTimeout, in that case i'll opt for creating a custom transaction